lib/app/views/main.html

```diff
@@ -10,47 +10,109 @@ <h1 ng-if="status.hasError">Error: {{status.error.name}}</h1>
       </a>
       <pre ng-if="status.hasError">{{status.error.message}}</pre>
     </div>
-    <input
-      class="sg sg-search-field"
-      id="search-section"
-      placeholder="Search styles"
-      ng-model="search.$"
-      type="search"
-      ng-model-options="{ updateOn: 'default blur', debounce: {'default': 500, 'blur': 0} }"
-      autofocus>
+    <div class="sg sg-search-container" ng-class="{ 'side-nav-search': config.data.sideNav }">
+      <div class="sg-hamburger " ng-click="toggleBool(toggleMenu)" ng-hide="!config.data.sideNav" ng-class="{ 'sg-hamburger-active': !toggleMenu }">
+        <div class="sg-hamburger-item top"></div>
+        <div class="sg-hamburger-item middle"></div>
+        <div class="sg-hamburger-item bottom"></div>
+      </div>
+      <input
+        class="sg sg-search-field"
+        id="search-section"
+        placeholder="Search styles"
+        ng-model="search.$"
+        type="search"
+        ng-model-options="{ updateOn: 'default blur', debounce: {'default': 500, 'blur': 0} }"
+        autofocus>
+    </div>
   </div>
 </header>
-<nav class="sg sg-top-nav" ng-class="{'designerToolVisible' : designerTool.isVisible}">
+
+<nav class="sg sg-top-nav" ng-class="{'designerToolVisible' : designerTool.isVisible}" ng-if="!config.data.sideNav">
   <ul class="sg sg-top-nav-menu">
     <li>
       <a class="sg"
         ng-class="currentSection === 'overview' ? 'active' : ''"
-        ng-click="clearSearch()"
+        ng-click="clearSearch();"
```

Comment:
If only one function call is used in ng-events, good practise is not to use semi-colon.